checkpoints/checkpoint3/script.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
```

Comment:
The speed jumps from 10 to 20, even though it seems that the starting speed is 20.